.github/workflows/test-coverage.yaml

```diff
@@ -246,3 +269,28 @@ jobs:
         with:
           path: ${{ github.event.repository.name }}
           allow-failure: ${{ inputs.allow-failure }}
+
+  publish-coverage-report:
+    name: Publish coverage report 📰
+    runs-on: ubuntu-latest
+    needs: coverage
+    # TODO Is it enough to filter out pull requests?
+    # Then the workflow will run on main and tags.
+    if: >
+      needs.coverage.outputs.publish-report == 'true'
+        && github.event_name != 'pull_request'
+    # Only one job can publish to gh-pages branch concurrently.
+    concurrency:
+      group: ghpages
+    steps:
+      - name: Download coverage report as artifact ⤵️
+        uses: actions/download-artifact@v3
+        with:
+          name: coverage-report
+          path: coverage-report
+      - name: Upload coverage report to GitHub pages 🗞️
```

Comment:
Is it ok to add a condition that would also copy the report to `latest-tag` in case the workflow is running for tag?